src/Stratis.Bitcoin/Base/ChainHeadersBehavior.cs

```diff
@@ -180,11 +180,13 @@ private void AttachedPeer_MessageReceived(NetworkPeer peer, IncomingMessage mess
                         // Use the ChainState.ConsensusTip property (not Chain.Tip)
                         // if the peer is behind/equal to our best height an empty array is sent back.
 
-                        if (!this.CanRespondToGetHeaders) break;
+                        if (!this.CanRespondToGetHeaders)
```

Comment:
we don't have such coding style policy, so this is just polluting the PR

Reply:
But it looks nicer with 'break' on the next line, isn't it? 
I can revert it if you want though.

Reply:
This is more of personal preference. So you prefer the break on the next line. I don't mind having either. But let's say if I preferred to have it on the same line. Then if I touched the code in next PR I would revert it, then if you touched the code you would revert what I reverted and so on :-) 

Therefore I tend not to fix anything that is just my personal preference and is not part of the code policy - because exactly we could play this ping pong all the time and it just pollutes the PR. So even if you think something is nicer, if it is not in code style, rather don't fix it.

Reply:
Reasonable. 
Fixed.